pandas/tests/test_frame.py

```diff
@@ -6320,6 +6319,20 @@ def test_to_csv_from_csv(self):
             xp.columns = lmap(int,xp.columns)
             assert_frame_equal(xp,rs)
 
+        with ensure_clean() as path:
+            # GH 10833 (TimedeltaIndex formatting)
+            dt = pd.Timedelta(seconds=1)
+            df_orig = pd.DataFrame({'data': list(range(10))},
+                                   index=[i*dt for i in range(10)])
+            df_orig.index.rename('timestamp', inplace=True)
```

Comment:
oh i c. if you want to trace where that is not passed thru would be gr8. (if its easy do it as part of this issue, if not, pls open a new issue).